R/derive_date_vars.R

```diff
@@ -530,8 +537,12 @@ restrict_imputed_dtc_dtm <- function(dtc,
         date_imputation = "last",
         time_imputation = "last"
       )
+    })
```

Comment:
I would not suppress all warning but just the one regarding `highest_imputation = "Y"`.

Reply:
Not sure I'm entirely following, I was trying to bake that into above, ~lines 312-314

Reply:
Sorry, I meant to add
```
regexpr = "If `highest_impuation` = \"Y\" is specified, `min_dates` or `max_dates` should be specified respectively."
```
Then we do not suppress _all_ warnings issued by the `impute_dtc_dtm()` calls but just the one regarding `highest_imputation = "Y"`.

Reply:
Could you specify the `regexpr` argument and add a comment why we suppress the warning?